KWZP2019/KWZP2019/EntranceControlForm.cs

```diff
@@ -502,7 +533,66 @@ private void ShowControlsFromDate(DateTime selectedDate)
                 orderBy.Data_dostarczenia)
                 .ToList();
             // Polish names only for displaying into dataGridView in form
-            dataGVEntranceControl.DataSource = orders;
+            dataGridViewEntranceControl.DataSource = orders;
+        }
+
+        // ==================================================
+
+        private void ShowControlsFromDate(int month, int year)
+        {
+            // var because it's an anonymouse type
+            var orders = db.SfOrderDetails
```

Comment:
zapytanie przenieść do bazy danych (widok), wtedy niepotrzebne będzie tworzenie obiektów w funkcji anonimowej

Reply:
Prawdopodobnie, nawet przy stworzonym widoku, i tak może być potrzeba tworzenia obiektów w funkcji anonimowej, jeżeli w tabeli miałby się wyświetlać kod półfabrykatu a w bazie operujemy na id półfabrykatu, taki model. 
Nawiasem mówiąc, ten kod już jest na masterze, więc nie wiem czemu komentarz dopiero teraz.

Reply:
1. należy stworzyć taki widok, który zwróci potrzebne dane (id i/lub kod półfabrykatu w zależności od potrzeb)
2. jeśli model jest zły to można go zmienić
3. a) PR zgłasza dodanie kodu tutaj
    b) to, że kod jest na masterze nie znaczy, że nie można go poprawić